src/app/components/input/input.component.ts

```diff
@@ -7,10 +10,10 @@ type InputLengthI = { title?: number, body?: number }
   templateUrl: './input.component.html',
   styleUrls: ['./input.component.scss']
 })
-export class InputComponent {
+export class InputComponent implements OnInit{
   showFirst = true;
   showDropdownMenu = false;
-
+  @Output() createNoteRefreshEvent = new EventEmitter<any>();
```

Comment:
basically the idea is we should have the @Input and the @Output first and then we should have the component variables it is recommended